tests/metrics/test_auc.py

```diff
@@ -277,3 +279,115 @@ def test_when_duplicates_in_interactions_insufficient(
             dtype=float,
         )
         pd.testing.assert_series_equal(metric.calc_per_user(reco, interactions), expected_metric_per_user)
+
+
+class TestDebiasableAUCMetric:
+    def setup_method(self) -> None:
+        self.reco = pd.DataFrame(
+            {
+                Columns.User: [1, 2, 3, 3, 3, 4, 5, 5, 5, 5],
+                Columns.Item: [1, 2, 1, 2, 3, 1, 1, 2, 3, 5],
+                Columns.Rank: [9, 1, 1, 2, 3, 1, 3, 7, 9, 1],
+            }
+        )
+        self.interactions = pd.DataFrame(
+            {
+                Columns.User: [1, 2, 3, 3, 3, 4, 5, 5, 5, 5],
+                Columns.Item: [1, 1, 1, 2, 3, 1, 1, 2, 3, 4],
+            }
+        )
+
+    @pytest.mark.parametrize(
+        "metric, metric_debias",
+        (
+            (
+                PartialAUC(k=1, insufficient_handling=InsufficientHandling.IGNORE),
+                PartialAUC(k=1, insufficient_handling=InsufficientHandling.IGNORE, debias_config=DEBIAS_CONFIG),
+            ),
+            (
+                PartialAUC(k=3, insufficient_handling=InsufficientHandling.IGNORE),
+                PartialAUC(k=3, insufficient_handling=InsufficientHandling.IGNORE, debias_config=DEBIAS_CONFIG),
+            ),
+            (
+                PartialAUC(k=1, insufficient_handling=InsufficientHandling.EXCLUDE),
+                PartialAUC(k=1, insufficient_handling=InsufficientHandling.EXCLUDE, debias_config=DEBIAS_CONFIG),
+            ),
+            (
+                PartialAUC(k=1, insufficient_handling=InsufficientHandling.EXCLUDE),
+                PartialAUC(k=1, insufficient_handling=InsufficientHandling.EXCLUDE, debias_config=DEBIAS_CONFIG),
+            ),
+            (
+                PAP(k=1, insufficient_handling=InsufficientHandling.IGNORE),
+                PAP(k=1, insufficient_handling=InsufficientHandling.IGNORE, debias_config=DEBIAS_CONFIG),
+            ),
+            (
+                PAP(k=3, insufficient_handling=InsufficientHandling.IGNORE),
+                PAP(k=3, insufficient_handling=InsufficientHandling.IGNORE, debias_config=DEBIAS_CONFIG),
+            ),
+            (
+                PAP(k=1, insufficient_handling=InsufficientHandling.EXCLUDE),
+                PAP(k=1, insufficient_handling=InsufficientHandling.EXCLUDE, debias_config=DEBIAS_CONFIG),
+            ),
+            (
+                PAP(k=3, insufficient_handling=InsufficientHandling.EXCLUDE),
+                PAP(k=3, insufficient_handling=InsufficientHandling.EXCLUDE, debias_config=DEBIAS_CONFIG),
+            ),
+        ),
+    )
+    def test_calc(self, metric: tp.Union[PartialAUC, PAP], metric_debias: tp.Union[PartialAUC, PAP]) -> None:
+        downsample_interactions = debias_interactions(self.interactions, config=metric_debias.debias_config)
```

Comment:
```suggestion
        debiased_interactions = debias_interactions(self.interactions, config=metric_debias.debias_config)
```

Reply:
(Everywhere)

Reply:
It may look like personal preferences, but actually it's really important to name your variables, functions, classes, etc. properly to make it easier to read and understand your code for other developers.

Here are 2 problems:
1. Grammatically incorrect name: "downsample interactions" is an imperative form, so it could be the name of function not variable. Here it should be "downsample**d** interactions".
2. Inconsistency: you call function `debias_interactions` over interactions, it's expected that in the result you should have "debiased interactions", not "downsampled". There may be exceptions from this rule, but here I don't see any reason for different naming.

Reply:
Sorry, my bad
Fixed